apis/v1/controllers/addresses_controller.js

```diff
@@ -214,14 +212,16 @@ exports.createAddress = async (req, res, next) => {
       } else {
         throw new Error("Invalide status");
       }
-
-    } else {
-      // addressData.status = statusMap.get(addressData.status);
-      addressData.status = 0;
-      throw new Error("User is not admin");
+    } if (platform === "common" && platform === "web") {
+      if (addressData !== undefined) {
```

Comment:
This test will always be true as you have defined the variable addressData and gave it a value. 
This could be a typo.
What exactly are you checking here?